editor/src/clj/editor/font.clj

```diff
@@ -701,7 +701,7 @@
             (dynamic edit-type (g/constantly (properties/->pb-choicebox Font$FontTextureFormat))))
   (property render-mode g/Keyword (default (protobuf/default Font$FontDesc :render-mode))
             (dynamic edit-type (g/constantly (properties/->pb-choicebox Font$FontRenderMode))))
-  (property size g/Int ; Required protobuf field.
+  (property size g/Int (default (protobuf/required-default Font$FontDesc :size))
```

Comment:
New concept: A required-default is the default value for a required protobuf field. We want the node to be constructed with this value if it hasn't been loaded, so we can't rely on the value being in the `pb-map`.